src/commands/functions-config-unset.ts

```diff
@@ -0,0 +1,46 @@
+import * as clc from "cli-color";
+
+import { Command } from "../command";
+import { logger } from "../logger";
+import { requirePermissions } from "../requirePermissions";
+import { needProjectId } from "../projectUtils";
+import * as functionsConfig from "../functionsConfig";
+import * as runtimeconfig from "../gcp/runtimeconfig";
+import * as utils from "../utils";
+
+export default new Command("functions:config:unset [keys...]")
+  .description("unset environment config at the specified path(s)")
+  .before(requirePermissions, [
+    "runtimeconfig.configs.list",
+    "runtimeconfig.configs.create",
+    "runtimeconfig.configs.get",
+    "runtimeconfig.configs.update",
+    "runtimeconfig.configs.delete",
+    "runtimeconfig.variables.list",
+    "runtimeconfig.variables.create",
+    "runtimeconfig.variables.get",
+    "runtimeconfig.variables.update",
+    "runtimeconfig.variables.delete",
+  ])
+  .before(functionsConfig.ensureApi)
+  .action(async (args, options) => {
+    if (!args.length) {
+      return utils.reject("Must supply at least one key");
```

Comment:
nit* Don't really care, but I'm guessing this could also be `new FirebaseError`

Reply:
Done.